doc/api/ffi.md

```diff
@@ -0,0 +1,132 @@
+# Foreign Function Interface (FFI)
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental
+
+<!-- source_link=lib/ffi.js -->
+
+The `node:ffi` module provides a foreign function interface (FFI). It can be
+used to make native function calls without the use of native addons. To access
+it:
+
+```mjs
+import ffi from 'node:ffi';
+```
+
+```cjs
+const ffi = require('node:ffi');
+```
+
+This module is only available under the `node:` scheme. The following will not
+work:
+
+```mjs
+import ffi from 'ffi';
+```
+
+```cjs
+const ffi = require('ffi');
```

Comment:
This should be `node:ffi`, and we need to ensure that the `node:` prefix is required so that this can be backported.